### ISSUE TITLE
Isolate Strategy implementation from the Framework

### ISSUE THREAD

**Issue by ramkumarkoppu** [chrism2671/PyTrendFollow#27]
It will be cleaner to move sample strategy implementation to it's own dir called TradeStrategy.

**Comment by chrism2671** [comment]
Yes, I agree it's not obvious how to create a strategy and test it, which is really the main strength of PyTrendFollow. It might make sense to start by making documentation on how a strategy should be created.
